distribution/docs/src/main/resources/content/_configuring/hiding-errors-by-role.adoc

```diff
@@ -19,5 +19,9 @@ If the *Metacard Validation Filter Plugin* is configured to *Filter errors* and/
 .. The default attribute for viewing invalid metacards is `invalid-state`
 ... `invalid-state=<USER ROLE>`.
 ... Replace `<USER ROLE>` with the roles that should be allowed to view invalid metacards.
+[NOTE]
+To prevent other DDF systems from querying invalid data in the local DDF catalog, it is
+suggested to create and set user roles that are unique to the local system (ie. a user role
```

Comment:
✏️ Nitpick: I think we usually use "recommended" (it is recommended to ...)

Reply:
I went ahead and changed it :)